lexicons/app/bsky/feed/post.json

```diff
@@ -38,6 +38,10 @@
             "type": "union",
             "refs": ["com.atproto.label.defs#selfLabels"]
           },
+          "tags": {
+            "type": "array",
```

Comment:
hashtags are notoriously stuffed for SEO/broadcast purposes, both as simple spam as as annoyingly "loud" promotion of otherwise legitimate content.

We *could* impost an array length limit here (8?). A quick search says that Tumblr has no limit but only indexes the first 5, which is maybe a simpler/better solution.

I'd be fine with a limit but could go either way. Not a blocking thing for PR, just in case it hadn't been considered.

Reply:
Not sure what's best here, but I would guess we _should_ add a limit e.g. we limit post length, it would be kinda silly if someone could add 100 tags or something.

Yeah maybe 8, or 10? Search can index differently if need be, right?

CC @pfrazee for thoughts

Reply:
Updated with a maxLength of 8

Reply:
8 sgtm